haskell/test-common/Lorentz/Contracts/Test/Common.hs

```diff
@@ -225,7 +225,7 @@ skipTest = do
     dummyContract = L.unpair L.# L.drop L.# L.nil L.# L.pair
   c <- lOriginate (L.defaultContract dummyContract) "skip test dummy" () (unsafeMkMutez 0)
   lCallEP c CallDefault ()
-  validate (Right expectAnySuccess)
+  expectNoUpdates
```

Comment:
I think `skipTest` is just not needed anymore. It should be possible to just do nothing now, after recent `Michelson.Test` changes.